### PR TITLE
PMM-8227 Another two metrics for pg_stat_monitor 0.9.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

File: agents/postgres/pgstatmonitor/models.go [hunk 1]
```diff
@@ -298,6 +302,10 @@ type pgStatMonitor struct {
 	PlansCalls        int64
 	WalFpi            int64
 	WalRecords        int64
+	WalBytes          int64
+	PlanTotalTime     float64
+	PlanMinTime       float64
+	PlanMaxTime       float64
```
Comment (BupycHuk) [inline agents/postgres/pgstatmonitor/models.go:308]: These fields are not passed

**Review by BupycHuk — APPROVED** [review]

File: agents/postgres/pgstatmonitor/pgstatmonitor.go [hunk 2]
```diff
@@ -260,6 +260,11 @@ func (m *PGStatMonitorQAN) makeBuckets(current, cache map[time.Time]map[string]*
 				},
 				Postgresql: new(agentpb.MetricsBucket_PostgreSQL),
 			}
+			if (currentPSM.PlanTotalTime - prevPSM.PlanTotalTime) != 0 {
+				mb.Postgresql.MPlanTimeSum = float32(currentPSM.PlanTotalTime-prevPSM.PlanTotalTime) / 1000
```
Comment (BupycHuk) [inline agents/postgres/pgstatmonitor/pgstatmonitor.go:264]: please add mb.Postgresql.MPlanTimeSum = count

[the remark this answers is not in the document]
Comment (BupycHuk) [reply agents/postgres/pgstatmonitor/pgstatmonitor.go:264]: please add a comment

Comment (percona-csalguero) [reply agents/postgres/pgstatmonitor/pgstatmonitor.go:264]: Done.

**Review by askomorokhov — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**